paper.tex

```diff
@@ -35,6 +35,8 @@
 
 \usepackage{longtable}
 
+\hyphenation{Math-e-ma-ti-ca pol-y-morph-ism}
```

Comment:
Yeah, that's why I added it. LaTeX thinks it's `Math-e-mat-ica`. I guess it depends on how you pronounce it. 

For Webster, I guess I'll use what they have. That's different from the pronounciation (you can see they have \ˌpä-lē-ˈmȯr-ˌfi-zəm\).  I've always thought of it as morph-ism, not mor-phism. 

Anyway, in both cases, that's the default hyphenation that LaTeX uses, and it causes overfull \hboxes. Maybe we should just rewrite the sentence.